django_db_logger/tests.py

```diff
@@ -22,25 +25,25 @@ def __test_log_aux(self, msg, fn, level):
         return log
 
     def test_log(self):
-        log = self.__test_log_aux('Info Message - 信息', self.logger.info, logging.INFO)
+        log = self.__test_log_aux('Info Message', self.logger.info, logging.INFO)
```

Comment:
Any reason why this has changed? I think we can revert the entire file

Reply:
Yes, you are right. It's reverted now.